content_sync/pipelines/definitions/concourse/site_pipeline.py

```diff
@@ -0,0 +1,656 @@
+        base.__init__(**kwargs)
+        self._config.site = site
+        self._config.pipeline_name = pipeline_name
+        self._config.is_root_website = is_root_website
+        self._config.purge_url = f"purge/{site.name}"
+        self._config.base_url = base_url
+        self._config.site_content_branch = site_content_branch
+        self._config.pipeline_name = pipeline_name
+        self._config.static_api_url = static_api_url
+        self._config.storage_bucket_name = storage_bucket_name
+        self._config.artifacts_bucket = artifacts_bucket
+        self._config.web_bucket = web_bucket
+        self._config.offline_bucket = offline_bucket
+        self._config.resource_base_url = resource_base_url
+        self._config.static_resources_subdirectory = static_resources_subdirectory
+        self._config.noindex = noindex
+        self._config.ocw_studio_url = ocw_studio_url
+        self._config.ocw_hugo_themes_branch = ocw_hugo_themes_branch
+        self._config.ocw_hugo_projects_url = ocw_hugo_projects_url
+        self._config.ocw_hugo_projects_branch = ocw_hugo_projects_branch
+        self._config.hugo_args_online = hugo_args_online
+        self._config.hugo_args_offline = hugo_args_offline
+        self._config.delete_flag = delete_flag
+        self._config.instance_vars = instance_vars
+        self._config.cli_endpoint_url = (
+            f" --endpoint-url {DEV_ENDPOINT_URL}" if is_dev() else ""
+        )
+
+        online_job = self.get_online_build_job()
+        offline_job = self.get_offline_build_job()
+        base.__init__(
+            resource_types=self.get_resource_types(),
+            resources=self.get_resources(),
+            jobs=[online_job, offline_job],
+            **kwargs,
+        )
+
+    def get_resource_types(self):
+        resource_types = [
+            self._http_resource_type,
+            self._keyval_resource_type,
+            self._s3_iam_resource_type,
+            slack_notification_resource(),
+        ]
+        return resource_types
+
+    def get_resources(self):
+        webpack_manifest_resource = Resource(
+            name=WEBPACK_MANIFEST_S3_IDENTIFIER,
+            type=self._s3_iam_resource_type.name,
+            check_every="never",
+            source={
+                "bucket": (self._config.artifacts_bucket or ""),
+                "versioned_file": f"ocw-hugo-themes/{self._config.ocw_hugo_themes_branch}/webpack.json",
+            },
+        )
+        if is_dev():
+            webpack_manifest_resource.source.update(
+                {
+                    "endpoint": DEV_ENDPOINT_URL,
+                    "access_key_id": (settings.AWS_ACCESS_KEY_ID or ""),
+                    "secret_access_key": (settings.AWS_SECRET_ACCESS_KEY or ""),
+                }
+            )
+        offline_build_gate_resource = Resource(
+            name=self._offline_build_gate_identifier,
+            type=self._keyval_resource_type.name,
+            check_every="never",
```

Comment:
My only question here is: "Are the `check_every` settings defined via this code consistent with the current `check_every` definitions across the existing jobs? This was a _major_ source of performance degradation. We just don't want to inadvertently revert that.

Reply:
Specifically w.r.t. the ocw_hugo_themes_resource, ocw_hugo_projects_resource and the other resources defined below.